frontend/packages/ceph-storage-plugin/src/utils/ocs-storage-class-params.tsx

```diff
@@ -90,6 +91,7 @@ export const StorageClassFormProvisoners: ExtensionSCProvisionerProp = Object.fr
           name: 'Encryption ID',
           hintText: 'A unique ID matching KMS ConfigMap',
           Component: StorageClassEncryptionKMSID,
+          visible: (params) => _.get(params, 'encrypted.value', false) === 'true',
```

Comment:
Prefer optional chaining.

```suggestion
          visible: (params) => params?.encrypted?.value === 'true',
```

Reply:
ack...